### PR TITLE
Avner

### PR DESCRIPTION
**Pull request by AvnerEisenberg**
added new file status panel

### CODE REVIEW

File: projects/ngx-composite-lib-tester/src/app/app.module.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { PepNgxCompositeLibModule } from '@pepperi-addons/ngx-composite-lib';
 import { PepColorSettingsModule } from '@pepperi-addons/ngx-composite-lib/color-settings';
 import { PepShadowSettingsModule } from '@pepperi-addons/ngx-composite-lib/shadow-settings';
 import { PepGroupButtonsSettingsModule } from '@pepperi-addons/ngx-composite-lib/group-buttons-settings';
-
+import { FileStatusPanelModule } from '@pepperi-addons/ngx-composite-lib/file-status-panel';
```
Comment (TomerPopik) [inline projects/ngx-composite-lib-tester/src/app/app.module.ts:14]: Add Pep prefix

File: projects/ngx-composite-lib/file-status-panel/file-status-panel.component.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit, Injectable, Input, Output, EventEmitter, Optional, Inject } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { IFile } from './file-status-panel.model';
+
+@Component({
+    selector: 'file-status-panel',
```
Comment (TomerPopik) [inline projects/ngx-composite-lib/file-status-panel/file-status-panel.component.ts:6]: Add pep- prefix

File: projects/ngx-composite-lib/file-status-panel/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "ngPackage": {
+        "lib": {
+            "entryFile": "public-api.ts",
+            "umdModuleIds": {
+                "@ngx-translate/core": "translate",
+                "@pepperi-addons/ngx-lib": "ngx-lib",
+                "@pepperi-addons/ngx-lib/field-title": "ngx-lib/field-title",
+                "@pepperi-addons/ngx-lib/icon": "ngx-lib/icon"
+            }
```
Comment (TomerPopik) [inline projects/ngx-composite-lib/file-status-panel/package.json:10]: Remove unused modules and add the the used modules.

File: projects/ngx-composite-lib/generic-list/generic-list.component.ts [hunk 4]
```diff
@@ -60,11 +60,17 @@ export class GenericListComponent implements OnInit, AfterViewInit {
     allowSelection: boolean = true;
   
     @Input()
-    noDataMessage: string = "No data";
+    noDataFoundMsg: string = this.translate.instant('GENERIC-LIST.NO_DATA');
```
Comment (TomerPopik) [inline projects/ngx-composite-lib/generic-list/generic-list.component.ts:63]: Change GENERIC-LIST.NO_DATA to GENERIC_LIST.NO_DATA (with underscore)

**Review by TomerPopik — CHANGES_REQUESTED** [review]

File: projects/ngx-composite-lib-tester/src/app/app.module.ts (hunk 1 above)
Comment (AvnerEisenberg) [reply projects/ngx-composite-lib-tester/src/app/app.module.ts]: done

File: projects/ngx-composite-lib/file-status-panel/file-status-panel.component.ts (hunk 2 above)
Comment (AvnerEisenberg) [reply projects/ngx-composite-lib/file-status-panel/file-status-panel.component.ts]: done

File: projects/ngx-composite-lib/file-status-panel/package.json (hunk 3 above)
Comment (AvnerEisenberg) [reply projects/ngx-composite-lib/file-status-panel/package.json:7]: done

File: projects/ngx-composite-lib/generic-list/generic-list.component.ts (hunk 4 above)
Comment (AvnerEisenberg) [reply projects/ngx-composite-lib/generic-list/generic-list.component.ts]: done

**Review by TomerPopik — APPROVED** [review]

**Pull request merged**